pkgs/development/interpreters/python/cpython/2.7/default.nix

```diff
@@ -335,8 +335,10 @@ in with passthru; stdenv.mkDerivation ({
       license = lib.licenses.psfl;
       platforms = lib.platforms.all;
       maintainers = with lib.maintainers; [ fridh ];
-      # Higher priority than Python 3.x so that `/bin/python` points to `/bin/python2`
-      # in case both 2 and 3 are installed.
-      priority = -100;
+      knownVulnerabilites = [
+        "CVE-2020-26116" # CVSS 7.2
+        "CVE-2020-27619" # CVSS 9.8
+        "CVE-2021-3177"  # CVSS 9.8
```

Comment:
This has a patch on the tree, might remove. The other two have been unpatched afaik.